events/blocks/event-map/event-map.js

```diff
@@ -1,10 +1,12 @@
 import { LIBS, getMetadata } from '../../scripts/utils.js';
 
 const { createTag } = await import(`${LIBS}/utils/utils.js`);
```

Comment:
These two imports would run a little faster if you did a await Promise.all([import 1.js, import 2.js]), and also if they are imported in the functions they are used instead of at top level.

Sorry I might have set bad examples of top level awaits in some other places, I'll be on my way to fix these 😅